cmd/otelcol/config/collector/gateway_config.yaml

```diff
@@ -142,6 +142,22 @@ exporters:
     token: "${SPLUNK_ACCESS_TOKEN}"
     endpoint: "${SPLUNK_INGEST_URL}/v1/log"
     log_data_enabled: false
+  # To send entities (applicable only if discovery mode is enabled)
+  otlphttp/entities:
```

Comment:
If this is only relevant when discovery is enabled, is there some way to tie this to discovery mode? (The pipeline is only enabled when discovery mode is enabled?)

Reply:
This is on the GW side and discovery might not be enabled there but enabled at the sender